### PR TITLE
Setup test server

### PR DESCRIPTION
**Pull request by RobinRockz**
testing server setup

### CODE REVIEW

**Review by laptopisasset — COMMENTED** [review]
I commented

File: client/src/App.tsx [hunk 1]
```diff
@@ -16,7 +16,7 @@ function App() {
           target="_blank"
           rel="noopener noreferrer"
         >
-          Learn React
+          Learn React!!!
```
Comment (laptopisasset) [inline client/src/App.tsx:19]: Please Remove the Extra !!!

**Review by laptopisasset — COMMENTED** [review]
Ok pls remove that!

File: client/src/App.tsx (hunk 1 above)
Comment (RobinRockz) [reply client/src/App.tsx:19]: Ok got it

**Review by laptopisasset — APPROVED** [review]
## Great Work

**Pull request merged**